versions.mk

```diff
@@ -3,16 +3,16 @@ KUBE_VER ?= v1.19.15
 SECCOMP_VER ?= 2.3.1-2.1+deb9u1
 DOCKER_VER ?= 20.10.7
 # we currently use our own flannel fork: gravitational/flannel
-FLANNEL_VER ?= v0.10.5-gravitational
-HELM_VER ?= 2.16.12
-HELM3_VER ?= 3.3.4
-COREDNS_VER ?= 1.7.0
-NODE_PROBLEM_DETECTOR_VER ?= v0.6.4
-CNI_VER ?= 0.8.6
+FLANNEL_VER := v0.10.5-gravitational
+HELM_VER := 2.16.12
+HELM3_VER := 3.3.4
+COREDNS_VER := 1.7.0
+NODE_PROBLEM_DETECTOR_VER := v0.6.4
+CNI_VER := 0.8.6
 SERF_VER ?= v0.8.5
-IPTABLES_VER ?= v1.8.5
```

Comment:
Whats up with these changes?  Are they ported from somewhere or do we no longer need to be able to override these? I ask because I don't see them in https://github.com/gravitational/planet/pull/871.

Reply:
These changes were picked up when cherry picking commit to master. I'll revert these changes since it probably makes more sense to allow variable override.